File: Models/Report.cs
using System;
using System.Collections.Generic;

namespace Accountant.Models
{
    public class Report
    {
        public int ID { get; set; }
        public ICollection<Expense> Expenses { get; set; }
        public bool Evaluated { get; set; }
        public DateTime DateOfEvaluation { get; set; }
    }
}

File: AccountantAndroid/app/src/main/java/com/kunkliricsi/accountant/model/Report.java
package com.kunkliricsi.accountant.model;

import java.util.Date;
import java.util.List;

public class Report {
    public int ID;
    public List<Expense> Expenses;
    public Date Start;
    public Date End;
    public boolean Evaluated;
    public Date DateOfEvaluation;
}
